libs/utils/src/lib/helpers/connections.ts

```diff
@@ -58,6 +58,19 @@ export const isSharedManagedConnector = (connectorId: string) => {
   return true;
 };
 
+export const isSharedManagedConnectorFromPApps = (connectorId: string) => {
```

Comment:
Ideally we shouldn't be coding for paths like this, we should expose some function to host which can tell if it is shared, the reason being, if we support another provider tomorrow then that also need be coded here instead of the host and this is not scalable.

Reply:
We should have like a resourceService that can determine if something is custom, armresource or sharedmanaged connector